resolwe/flow/models/data.py

```diff
@@ -544,6 +544,11 @@ def save(self, render_name=False, *args, **kwargs):
                 self.input, self.process.slug, self.process.version
             )
 
+            validate_schema(
```

Comment:
Why is this not in a single line?

Reply:
Most probably my fault since I forgot to remove the last comma and let black handle the formatting.